.changeset/tiny-glasses-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/dev-cli": patch
+---
+
+Adjust package.json to align with the other published packages inside clerk/javascript
```

Comment:
```suggestion
---
---
```
I think we don't need to release a version for these changes

Reply:
But otherwise they wouldn't get published (which is the intention) 😅

Reply:
Ok!